frontend/src/components/App/Notifications/index.tsx

```diff
@@ -228,6 +252,35 @@ export default function Notifications() {
         }}
         id={notificationMenuId}
       >
+        <Box borderBottom={`1px solid ${theme.palette.notificationBorderColor}`} p={1}>
+          <Grid container justifyContent="space-between">
+            <Grid item>
+              <Box mx={1}>
+                <Typography style={{ fontWeight: 'bold' }}>{t('Notifications')}</Typography>
+              </Box>
+            </Grid>
+            <Grid item>
+              <Box display={'flex'} justifyContent="space-between">
+                <Box px={1}>
+                  <Link
```

Comment:
All features should be usable with the keyboard. If you try tabbing through, it's not possible to use the keyboard for these links (I think changing them to buttons would fix it).